app/src/main/java/com/xmartlabs/template/service/TaskService.kt

```diff
@@ -0,0 +1,26 @@
+package com.xmartlabs.template.service
+
+import com.xmartlabs.template.model.Balance
+import com.xmartlabs.template.model.Entry
+import com.xmartlabs.template.model.EntryRequest
+import com.xmartlabs.template.model.EntryResponse
+import com.xmartlabs.template.model.User
+import io.reactivex.Single
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+interface TaskService {
+  @GET("/tasks/1/users/{user_id}/entries")
```

Comment:
i thinl it would be better to use constants instead of pure strings for this since tomorrow a name changes and it just change one constant